starter-core/src/main/java/io/micronaut/starter/feature/function/gcp/template/gcpFunctionSpock.rocker.raw

```diff
@@ -19,7 +19,18 @@ class @project.getClassName()FunctionSpec extends Specification {
 
     void "test function"() {
         when:"The function is executed"
-        GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/@project.getPropertyName()")
+        GoogleHttpResponse response = function.invoke(HttpMethod.GET, "/")
+
+        then:"The response is correct"
+        response.status == HttpStatus.OK
+    }
+
+    void "test post function"() {
+        when:"The POST function is executed"
+        SampleInputMessage sampleInputMessage = new SampleInputMessage()
+        sampleInputMessage.setName("Test Name")
+        HttpRequest request = HttpRequest.POST("/", sampleInputMessage).contentType(MediaType.APPLICATION_JSON_TYPE)
```

Comment:
`.contentType(MediaType.APPLICATION_JSON_TYPE)` is unnecessary. Micronaut defaults to `application/json`